e2e/link_js_package/lib/BUILD.bazel

```diff
@@ -0,0 +1,11 @@
+load("@aspect_rules_js//js:js_package.bzl", "js_package")
+
+js_package(
+    name = "lib",
+    srcs = [
```

Comment:
this is using `copy_to_directory` impl under the hood and inherits all of its attributes and features